src/sed.js

```diff
@@ -22,6 +22,9 @@ function _sed(options, regex, replacement, files) {
     'i': 'inplace'
   });
 
+  // Check if this is coming from a pipe
+  var pipe = common.readFromPipe(this);
```

Comment:
Hmmm... not sure if that would be any cleaner. I thought about prepending `'-'` to the list of files inside `common.wrap()`, but individual utilities would still need to check for the piped input, so it wouldn't save us much.